app/exchangers/ekylibre/settings_exchanger.rb

```diff
@@ -140,9 +140,12 @@ def import
 
       # Load accounts
       if can_load_default?(:accounts)
+        # Account number can't start with a '0' and are 8 caracters length
+        min_account_number = 10000000
+        max_account_number = 99999999
         @manifest[:accounts] = Cash.nature.values.each_with_object({}) do |nature, hash|
           hash[nature] = { name: "enumerize.cash.nature.#{nature}".t,
-                           number: format('%08d', rand(10**7)) }
+                           number: format('%08d', rand(min_account_number..max_account_number)) }
```

Comment:
Shouldn't be random:
 - Cash nature Bank Account         => 512
 - Cash nature Cash Box                => 53
 - Cash nature Associate Account => 455